csplit/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "csplit"
+version = "0.1.0"
+authors = ["Mats Kindahl <mats.kindahl@gmail.com>"]
+build = "build.rs"
+edition = "2018"
+
+[dependencies]
+clap = { version = "^2.33.0", features = ["yaml", "wrap_help"] }
+regex = "1.3.1"
```

Comment:
Please, allow other versions of regex using the `^` at the start of the version number

Reply:
Done.